tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "noImplicitAny": false,
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
```

Comment:
I don't think you need `allowSyntheticDefaultImports` when `esModuleInterop` is true

Reply:
Great, thanks!